hw/ip/rom_ctrl/dv/cov/rom_ctrl_cov_excl.el

```diff
@@ -146,6 +146,135 @@ Condition 37 "2164803938" "(tl_i_int.a_valid & reqfifo_wready & ((~error_interna
+// conditional statement explains why.
+INSTANCE: tb.dut.u_tl_adapter_rom.u_reqfifo
+Condition 3 "1324655787" "(rvalid_o & rready_i & ((~gen_normal_fifo.under_rst))) 1 -1" (3 "110")
+
+// For u_rspfifo, Pass=1 and rvalid_o is (fifo_empty & ~wvalid_i) & ~under_rst. wvalid_i comes from
+// u_tl_adapter_rom as rspfifo_wvalid. rspfifo_wvalid needs both rvalid_i (response from rom) and
+// reqfifo_rvalid. But as explained above that we can't get rvalid_o when under_rst=1. This gets
+// the (fifo_empty & ~wvalid_i) term true which means empty=1. Since to cover 110, we need
+// under_rst=1 which then negates to 0 inside the assignment leads to no rvalid_o. So, same
+// conclusion as u_reqfifo that we can't get rvalid_o when under_rst=1.
+//
+// For rready_i, it comes in as rspfifo_rready from u_tl_adapter_rom through a conditional
+// assignment. If the condition is true, reqfifo_rready gets assigned otherwise 1'b0 is assigned
+// to rspfifo_rready. In either case, rspfifo_rready should be false as when under_rst=1, we can't
+// get rvalid_o and with no rvalid_o there shouldn't be any rready_i in case of u_reqfifo.
+INSTANCE: tb.dut.u_tl_adapter_rom.u_rspfifo
+Condition 3 "1324655787" "(rvalid_o & rready_i & ((~gen_normal_fifo.under_rst))) 1 -1" (3 "110")
+
+// For u_sramreqfifo, the reasoning is similar as u_reqfifo i.e when under_rst=1 we can't get
+// rvalid_o. For rready_i, it comes in from u_tl_adapter_rom and gets assigned by rspfifo_wvalid.
+// As explained above in the case of u_rspfifo about how to get rspfifo_wvalid which requires
+// reqfifo_rvalid true to set itself true which is impossible if under_rst=1
+INSTANCE: tb.dut.u_tl_adapter_rom.u_sramreqfifo
+Condition 3 "1324655787" "(rvalid_o & rready_i & ((~gen_normal_fifo.under_rst))) 1 -1" (3 "110")
+
+// The case 101 for the conditional statements below is non-occuring.
+//
+// For u_reqfifo, if wready_o=0 this means that the fifo is either in reset or full. wvalid_i comes
+// from u_tl_adapter_rom as reqfifo_wvalid. It gets continuous assignment through a_ack. a_ack is
+// true if we get a_valid coming from u_sram_byte as one of the ports of tl_i_int and a_ready going
+// into u_sram_byte as one of the ports of tl_o_int. In order to set tl_o_int.a_ready as true, we
+// need reqfifo_wready as true. Hence, if we don't have wready_o as true, we can't get
+// tl_o_int.a_ready in u_tl_adapter_rom which means that a_ack should be false leading to wvalid_i
+// being false.
+INSTANCE: tb.dut.u_tl_adapter_rom.u_reqfifo
+Condition 3 "786039886" "(wvalid_i & wready_o & ((~gen_normal_fifo.under_rst))) 1 -1" (2 "101")
+
+// For u_rspfifo, wvalid_i comes from u_tl_adapter_rom as rspfifo_wvalid. In order to set it true,
+// we need reqfifo_rvalid true. Which means that we are reading the write that happened earlier and
+// u_rspfifo needs to respond to that. If u_rspfifo is responding now then it means that it is not
+// full and we can have wready_o with wvalid_i.
```

Comment:
This took me a few seconds to properly understand, and I think that it could be explained more clearly (for me, at least!). Something like this, maybe?
```systemverilog
// The u_rspfifo instance in tlul_adapter_sram can never give any backpressure through the wready_o
// signal. This is because u_reqfifo will always be at least as full as u_rspfifo, each item in
// u_rspfifo is a response to a request that is still stored in u_reqfifo. The request is stored in
// order to be able to supply d_size and d_source in the TL response.
//
// If w_valid_i is true in u_rspfifo, this is the response to some request which must be stored in
// u_reqfifo, which implies there is a slot to store it.
```